controllers/storageclassclaim_controller.go

```diff
@@ -572,14 +578,18 @@ func (r *StorageClassClaimReconciler) own(resource metav1.Object) error {
 	return controllerutil.SetOwnerReference(r.storageClassClaim, resource, r.Scheme)
 }
 
-// addAnnotation add a annotation to a resource metadata
-func addAnnotation(obj metav1.Object, key string, value string) {
+// addAnnotation adds an annotation to a resource metadata, returns true if added
```

Comment:
```suggestion
// addAnnotation adds an annotation to a resource metadata, returns true if added else false
```

Reply:
well, return value can't be empty and bool has only two valid values, nevertheless took in the suggestion as being explicit doesn't hut.